### DISCUSSION TITLE
Initializing the database

### DISCUSSION THREAD

**Discussion by myFinTechPL** [talkpython/htmx-django-course#2] [Q&A] [answered]
## First run error - missing database
I've got an error after trying to follow the ch3/8 "Creating a PyCharm project" and the page was not loading. I realized that there was an empty database. I have run the following commands after finding load.py file:
1. python manage.py migrate
2. python manage.py load misc\videos.json

I have not watched the rest of the course yet, but I think we are missing instructions here to properly initialize and load the data into the database. Please advise.

**Comment by cltrudeau** [comment, marked as answer]
Hi myFinTechPL,

The SQLite databases were meant to be included in the repo. I used a standard .gitignore file which of course included configuration to ignore database files.

It has been addressed. Sorry about any trouble.

**Comment by mikeckennedy** [comment]
Thank you @cltrudeau!
